app/controllers/services_controller.rb

```diff
@@ -14,6 +14,7 @@ def edit
 
   def update
     if @bibliography_service.update(update_params)
+      @exhibit.sync_bibliography if @bibliography_service.api_settings_changed?
```

Comment:
I would be 👌 with `update_and_sync_bibliography`.  I'll make the change unless @mejackreed beats me to it.